tests/unittests/OperatorTest.cpp

```diff
@@ -3734,6 +3734,55 @@ TEST_P(InterpAndCPU, SparseLengthsSum) {
   EXPECT_TRUE(expected.isEqual(result));
 }
 
+TEST_P(InterpOnly, SparseLengthsSumI8) {
+  /*
+    DATA  = [
+        [1.0, 1.2],
```

Comment:
this does not match to what we do ;)

Reply:
I think I just used the dequantized values, but fair enough, will fix.